.github/workflows/tests/test_stac.py

```diff
@@ -1,8 +1,19 @@
 """test veda-backend STAC."""
 
+import json
+
 import httpx
+from openapi_schema_validator import validate
+
+with open(".github/workflows/tests/schemas/feature_schema.json", "r") as f:
```

Comment:
Can you the published STAC collection and features schemas instead of creating our own here?

Reply:
Yes, working on that for the next commit